go.mod

```diff
@@ -18,37 +18,37 @@ require (
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20191113165036-4c7a9d0fe056
-	google.golang.org/grpc v1.25.1
-	k8s.io/api v0.17.5
-	k8s.io/apimachinery v0.17.5
-	k8s.io/client-go v0.17.5
+	google.golang.org/grpc v1.26.0
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
+	k8s.io/client-go v0.18.3
 	k8s.io/kubernetes v1.16.2
-	k8s.io/utils v0.0.0-20191218082557-f07c713de883
+	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.17.5
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.5
-	k8s.io/apimachinery => k8s.io/apimachinery v0.17.6-beta.0
-	k8s.io/apiserver => k8s.io/apiserver v0.17.5
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.5
-	k8s.io/client-go => k8s.io/client-go v0.17.5
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.5
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.5
-	k8s.io/code-generator => k8s.io/code-generator v0.17.6-beta.0
-	k8s.io/component-base => k8s.io/component-base v0.17.5
-	k8s.io/cri-api => k8s.io/cri-api v0.17.6-beta.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.5
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.5
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.5
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.5
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.5
-	k8s.io/kubectl => k8s.io/kubectl v0.17.5
-	k8s.io/kubelet => k8s.io/kubelet v0.17.5
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.5
-	k8s.io/metrics => k8s.io/metrics v0.17.5
+	k8s.io/api => k8s.io/api v0.18.3
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.3
+	k8s.io/apimachinery => k8s.io/apimachinery v0.18.4-rc.0
```

Comment:
Is this intentionally not `0.18.3`?

Reply:
This one's a bit odd: our upgrade script puts in a replacement rule to v0.18.3 properly. It's only when `go mod tidy` runs that k8s.io/apimachinery is replaced for v0.18.4-rc.0.

I played around with `go mod why` and `go mod graph` but couldn't find any indicators for why the release candidate would be referenced. I wonder if this is a bug. 🤔 

On the bright side, changing the replacement to v0.18.3 and running `go mod tidy` afterwards seems to work in the sense that the rc version is _not_ replaced again. That'd work for me for now unless you may have another idea on what we can or should do here.

Reply:
I think I understand now:  k8s.io/apimachinery@v0.18.3 and k8s.io/apimachinery@v0.18.4-rc.0 point at the same commit. It seems similar with a few other dependencies.

I forced all of them into v0.18.3, it makes sense to me to be consistent.